pycaption/scc/specialized_collections.py

```diff
@@ -1,9 +1,23 @@
-from ..base import Caption, CaptionNode
+from ..base import CaptionList, Caption, CaptionNode
 from ..geometry import (UnitEnum, Size, Layout, Point, Alignment,
                         VerticalAlignmentEnum, HorizontalAlignmentEnum)
 
 from .constants import PAC_BYTES_TO_POSITIONING_MAP, COMMANDS
 
+class PreCaption(object):
```

Comment:
A docstring would be awesome. Of course we know the idea: Captions are now considered immutable, so we have this thing, mutate it around, and then commit this to a caption, but other people might wonder for half an hour why we need this.